core/templates/pages/topic-editor-page/editor-tab/topic-editor-tab.directive.ts

```diff
@@ -236,8 +239,11 @@ export class TopicEditorTabComponent implements OnInit, OnDestroy {
     );
   }
 
-  getClassroomUrlFragment(): string {
-    return this.topicEditorStateService.getClassroomUrlFragment();
+  isAssignedToAClassroom(): boolean {
+    if (this.classroomUrlFragment && this.classroomName) {
```

Comment:
"return this.classroomUrlFragment && this.classroomName" is good enough

Reply:
Done.

Reply:
Done.